src/pybind/mgr/nfs/export.py

```diff
@@ -76,6 +77,15 @@ def validate_cephfs_path(mgr: 'Module', fs_name: str, path: str) -> None:
         raise NFSException(e.args[1], -e.args[0])
 
 
+def _validate_cmount_path(cmount_path: str, path: str) -> None:
+    if cmount_path not in path:
```

Comment:
so now there is no path validation of cmount_path, i dont think `if cmount_path not in path` can be counted as one.

Reply:
Well `path` is already validated which is ofc now enough  to  test with this if condition

Reply:
so you mean cmount_path here is validated?

Reply:
yes, if `path` is already validated it'll suffice with the if condition I added!